src/Store/Actions/PortfolioActions.js

```diff
@@ -40,7 +40,12 @@ export const searchPortfolioItems = value => ({
     })
 });
 
-export const addPortfolioWithItem = (portfolioData, itemData) => ({
+export const addPortfolio = (portfolioData, items) => ({
     type: ActionTypes.ADD_PORTFOLIO,
-    payload: PortfolioHelper.addPortfolioWithItem(portfolioData, itemData)
+    payload: PortfolioHelper.addPortfolio(portfolioData, items)
+});
+
+export const updatePortfolio = (portfolioData) => ({
+    type: ActionTypes.UPDATE_PORTFOLIO,
```

Comment:
You are missing UPDATE_PORTFOLIO in ActionTypes.

Reply:
The API for this does not exist yet - I will add the Action type when the API is actually available.

Reply:
No worries, I already added the type in my actual open PR => an ugly warning disappeared and everything works as before.